pyfr/denserkschemes/rkopt/vermeire-loppi-vincent-s7-p0.txt

```diff
@@ -0,0 +1,7 @@
+0.2328811281838825 0.0 0.0 0.0 0.0 0.0 0.0
```

Comment:
Strip the 0's (don't want to encourage people to try and write out an implicit scheme!)